plugin/bufstop.vim

```diff
@@ -179,9 +183,17 @@ function! s:BufstopWipeBuffer(bufnr)
   setlocal nomodifiable
 endfunction
 
+function! BufstopRestoreWinview()
+  q
+  if exists('b:bufstop_winview')
+    call winrestview(b:bufstop_winview)
+    unlet b:bufstop_winview
+  endif
+endfunction
```

Comment:
You need to return to the previous window here (`wincmd p`), othewise your code won't work when two or more windows are present (split windows).

Reply:
Ah I see, between `endif` and `endfunction` right? for single window (not using split window), there's no problem?

Reply:
yes, this should work